sample/src/main/java/com/robotemi/sdk/sample/MainActivity.kt

```diff
@@ -339,6 +339,8 @@ class MainActivity : AppCompatActivity(), NlpListener, OnRobotReadyListener,
         btnPublish.setOnClickListener { publishToActivityStream() }
         btnHideTopBar.setOnClickListener { hideTopBar() }
         btnShowTopBar.setOnClickListener { showTopBar() }
+        btnWakeup.setOnClickListener { wakeup() }
```

Comment:
I would suggest to add another button or something speicial to do it.

wakeup() can still be the default `robot.wakeup()`